OpenRA.Game/HotkeyModifiers.cs

```diff
@@ -0,0 +1,25 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+namespace OpenRA
+{
+	public class HotkeyModifiers
```

Comment:
`Renderer` already handles this stuff internally and does expose some public input related state (`GetClipboardText`/`SetClipboardText`). I'm not necessarily suggesting that we expose `Renderer.HasModifier` instead, but i wonder if we're missing a more optimal abstraction that covers both.

Reply:
How about we move the `platform.CreateWindow` call into `Game.Initialize` and then pass that to the `Renderer` ctor plus a new global `Input`? We can then remove the `PumpInput` call from `Renderer.EndFrame` and have that as its own thing in `Input`.

Reply:
Dropping these changes here, will submit a new pr with changes.